tests/test_transforms/test_inference/conftest.py

```diff
@@ -60,7 +60,9 @@ def ts_with_external_timestamp(regular_ts) -> TSDataset:
     df_exog = df.copy()
     df_exog["external_timestamp"] = df["timestamp"]
     df_exog.drop(columns=["target"], inplace=True)
-    ts = TSDataset(df=TSDataset.to_dataset(df).iloc[5:], df_exog=TSDataset.to_dataset(df_exog), freq="D")
+    ts = TSDataset(
+        df=TSDataset.to_dataset(df).iloc[5:-1], df_exog=TSDataset.to_dataset(df_exog), freq="D", known_future="all"
```

Comment:
Why are we here excluding a last timestamp?

Reply:
I excluded it to make `external_timestamp` a regressor. Otherwise setting `known_future="all" will lead to error.